standalone/src/models/cart/cart/line-item/constants.ts

```diff
@@ -1,16 +1,11 @@
-export const inventoryMode = {
-  TrackOnly: 'TrackOnly',
-  ReserveOnOrder: 'ReserveOnOrder',
-  None: 'None',
-} as const;
+import {
```

Comment:
Instead of using raw string values, we now export the GraphQL generated TS enums so we can populate the properties with the actual expected types.
REST model can also use the string values of each enum item.